Samples/SqueezeNetObjectDetection/NET5/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "SqueezeNetObjectDetectionNET5": {
+      "commandName": "Project",
+      "commandLineArgs": "SqueezeNet.onnx kitten_224.png cpu",

```

Comment:
do we do this in other samples? I think I like it, so it shows something useful right away. maybe we should add it elsewhere.

Reply:
yeah, until now this file was part of the gitignore. We should add the relevant copy commands and launchSettings to each sample